data/_models/shared/Processes.csv

```diff
@@ -1,51 +1,104 @@
+Experiments,,sample preparation,heading,,,,,,,,,,,Patient registry
+Experiments,,input amount,int,,,,,,,,,http://purl.allotrope.org/ontologies/role#AFRL_0000010,Amount of input material in nanogram (ng).,"Patient registry, FAIR Genomes"
+Experiments,,library preparation kit,ontology,,,CatalogueOntologies,Sequencing enrichment kits,,,,,http://purl.obolibrary.org/obo/GENEPIO_0000085,"Pre-filled, ready-to-use reagent cartridges intented to improve chemistry, cluster density and read length as well as improve quality (Q) scores for this sample. Reagent components are encoded to interact with the sequencing system to validate compatibility with user-defined applications.","Patient registry, FAIR Genomes"
+Experiments,,PCR free,bool,,,,,,,,,http://purl.obolibrary.org/obo/NCIT_C17003,Indicates whether a polymerase chain reaction (PCR) was used to prepare this sample. PCR is a method for amplifying a DNA base sequence using multiple rounds of heat denaturation of the DNA and annealing of oligonucleotide primers complementary to flanking regions in the presence of a heat-stable polymerase.,"Patient registry, FAIR Genomes"
+Experiments,,target enrichment kit,ontology,,,CatalogueOntologies,Sequencing enrichment kits,,,,,http://purl.obolibrary.org/obo/NCIT_C154307,Indicates which target enrichment kit was used to prepare this sample. Target enrichment is a pre-sequencing DNA preparation step where DNA sequences are either directly amplified (amplicon or multiplex PCR-based) or captured (hybrid capture-based) in order to only focus on specific regions of a genome or DNA sample.,"Patient registry, FAIR Genomes"
+Experiments,,UMIs present,bool,,,,,,,,,http://www.ebi.ac.uk/efo/EFO_0010199,Indicates whether any unique molecular identifiers (UMIs) are present. An UMI barcode is a short nucleotide sequence that is used to identify reads originating from an individual mRNA molecule.,"Patient registry, FAIR Genomes"
+Experiments,,intended insert size,int,,,,,,,,,https://w3id.org/fair-genomes/resource/FG_0000001,"In paired-end sequencing, the DNA between the adapter sequences is the insert. The length of this sequence is known as the insert size, not to be confused with the inner distance between reads. So, fragment length equals read adapter length (2x) plus insert size, and insert size equals read lenght (2x) plus inner distance.","Patient registry, FAIR Genomes"
+Experiments,,intended read length,int,,,,,,,,,http://purl.obolibrary.org/obo/NCIT_C153362,The number of nucleotides intended to be ordered from each side of a nucleic acid fragment obtained after the completion of a sequencing process.,"Patient registry, FAIR Genomes"
+Experiments,,barcode,string,,,,,,,,,http://purl.obolibrary.org/obo/NCIT_C43361,A machine-readable representation of information in a visual format on a surface.,Patient registry
+Experiments OGM,Experiments,,,,,,,,,,,"http://purl.obolibrary.org/obo/NCIT_C116162",An Optical Genome Mapping (OGM) experiment,Patient registry
```

Comment:
nice. Assuming we would change emx2 so that subclass tables are shown directly underneath its parent, would it then be nicer to call these 'OGM Experiments', 'RNA experiments' etc?

Then I envision in table list view:
- Experiments
  - OGM Experiments
  - RNA Experiments
  - etc